### PR TITLE
Update filter kit

### PR DESCRIPTION
**Pull request by xjunior**
#### Runway Ticket URL

Updates Filter kit to the version used in Nitro, with ransack compatible API.

screencapture-localhost-8080-kits-filter-react-2020-05-19-22_33_48

Also fixes https://github.com/powerhome/playbook/pull/790

### CODE REVIEW

File: app/pb_kits/playbook/pb_filter/docs/_filter_default.jsx [hunk 1]
```diff
@@ -11,16 +11,20 @@ const FilterDefault = () => {
...
-      { item: 'Title', link: '#', active: false },
-      { item: 'Name', link: '#', active: false },
-    ]}
+        sortOptions={{
+          popularity: 'Popularity',
+          // eslint-disable-next-line
+          manager_title: 'Manager\'s Title',
+          // eslint-disable-next-line
+          manager_name: 'Manager\'s Name',
+        }}
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_filter/docs/_filter_default.jsx:26]: Can we update the rails docs to they are in sync (Added "manager's")?

**Comment by jasoncypret** [comment]
Added the Milano tag. We have deploys now!

**Comment by kre8sions** [comment]
@xjunior one update can we revert the change on the results for the double variant to use the same bold text as the rails version.
Screen Shot 2020-05-20 at 5 08 18 PM

**Comment by xjunior** [comment]
Done, @kre8sions

**Pull request merged**